src/aap_eda/core/management/commands/create_initial_data.py

```diff
@@ -913,6 +914,45 @@
     ]
 }
 
+ANALYTICS_CREDENTIAL_INPUTS = {
+    "fields": [
+        {
+            "id": "analytics_url",
```

Comment:
This field is not provided by customer, but from installer.

We need to add field for INSIGHTS_TRACKING_STATE, which is different from the feature flag.